include/aspect/initial_conditions/S40RTS_perturbation.h

```diff
@@ -31,7 +31,7 @@ namespace aspect
   {
     using namespace dealii;
 
-    namespace internal
+    namespace S40RTSinternal
```

Comment:
Why is this change necessary? Do you get linker errors because you declare the same classes in your own model?

In that case, my preference would be to use
```
namespace internal
{
  namespace S40RTS
  {
    class SphericalHarmonics;
```
and similar, but with a different *inner* namespace in your own model.

Of course, this begs the question: are the classes you declare actually different and just happen to have the same name? Or do you simply use a copy?